integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/DashboardWomenSpecification.groovy

```diff
@@ -162,6 +162,104 @@ class DashboardWomenSpecification extends AbstractSpecification {
       'Total 8']
   }
 
+
+  def "The initial cat dashboard should show correct women YOI stats data"() {
```

Comment:
Do we need additional test scenarios like this one "The initial cat dashboard should show correct women YOI stats data" and "The recat dashboard should show correct women YOI stats data"?
The more tests we add here the slower it gets to run the full suite.
Other tests e.g. "The recat dashboard should show correct change table for Women"  have been now extended with YOI categories, is that sufficient?